package.json

```diff
@@ -57,7 +57,8 @@
     "decanter": "^7.2.0",
     "eslint": "^8.50.0",
     "eslint-config-next": "^13.4.13",
```

Comment:
If we could update eslint-config-next to match the version of Next.js we're using that'd be awesome 😄

Reply:
Good catch. Thanks.